test/Microsoft.VisualStudio.Composition.Tests/StaticMemberExportsTests.cs

```diff
@@ -262,7 +268,7 @@ public void StaticPropertyExportDoesNotInstantiateClass(IContainer container)
             Assert.False(ClassWithStaticMemberExports.ConstructorCalled, "Constructor should not be called for static property export");
         }
 
-        [MefFact(CompositionEngines.V3EmulatingV1, typeof(ClassWithStaticMemberExports))]
+        [MefFact(CompositionEngines.V1Compat | CompositionEngines.V2Compat | CompositionEngines.V3EmulatingV1 | CompositionEngines.V3EmulatingV2, typeof(ClassWithStaticMemberExports))]
```

Comment:
V1Compat includes V3EmulatingV1, and V2Compat includes V3EmulatingV2.
Please drop the redundant flags.